securesystemslib/_vendor/test-ed25519-upstream.sh

```diff
@@ -12,13 +12,12 @@ set -eu
 # This commit matches our securesystemslib/_vendor/ed25519/ content.
 # If upstream changes, we should review the changes, vendor them,
 # and update the hash here
-pyca_ed25519_expected="c13748e1d24c5c00f6ce2b9c38a319ae02355d97"
+pyca_ed25519_expected="0abd61bd0ef914133906b797ad568b35d44744a2"
 pyca_ed25519_git_url="https://github.com/pyca/ed25519.git"
 
 pyca_ed25519_master_head=$(git ls-remote "$pyca_ed25519_git_url" master | cut -f1)
 if [ "$pyca_ed25519_master_head" != "$pyca_ed25519_expected" ]; then
-    echo "$pyca_ed25519_git_url master branch has been updated." >&2
-    echo "Expected $pyca_ed25519_expected, found $pyca_ed25519_master_head." >&2
+    echo "Expected commit ${pyca_ed25519_expected:0:7}, found ${pyca_ed25519_master_head:0:7}."
```

Comment:
Would be cool to make the commits clickable in the GitHub issue...
```
pyca/ed25519@<commit id>
```
... and maybe even add a link to a compare view...
```
https://github.com/pyca/ed25519/compare/<commit id>...<commit id>
```

The workflow file might be the "right" place to add these GitHub specifics, but it seems a lot easier to just add it here. :)